testing/xerces-c/APKBUILD

```
@@ -0,0 +1,41 @@
+# Contributor: Andrew Bell <andrew.bell.ia@gmail.com>
+# Maintainer: Andrew Bell <andrew.bell.ia@gmail.com>
+pkgname=xerces-c
+pkgver=3.2.2
+pkgrel=0
+pkgdesc="Xerces-C++ is a validating XML parser written in a portable subset of C++."
```

You can leave out the project name from the description, so something like

```suggestion
pkgdesc="A validating XML parser written in a portable subset of C++."
```